--- old
+++ new
@@ -1,1 +1,1 @@
-= Babun - a windows shell you will love
+# Babun - оболочка Windows, которая вам понравится
@@ -3,1 +3,3 @@
-Would you like to use a linux-like console on a Windows host without a lot of fuzz? Try out babun!
+Хотели бы вы использовать консоль, подобную Linux, на хосте Windows без особых сложностей? Попробуй бабуна!
+
+```
@@ -16,0 +18,1 @@
+```
@@ -17,1 +20,0 @@
-
@@ -28,1 +30,1 @@
-== Installation
+## Installation
@@ -30,2 +32,2 @@
-Just download the dist file from http://babun.github.io, unzip it and run the install.bat script. After a few minutes babun starts automatically.
-The application will be installed to the `%USER_HOME%\.babun` directory. Use the '/target' option to install babun to a custom directory.
+Просто скачайте файл dist с http://babun.github.io , распакуйте его и запустите скрипт install.bat. Через несколько минут babun запускается автоматически.
+Приложение будет установлено в каталог `%USER_HOME%\.babun`. Используйте опцию '/target', чтобы установить babun в пользовательский каталог.
@@ -33,1 +35,1 @@
-NOTE: There is no interference with existing Cygwin installation
+NOTE: Нет никаких помех существующей установке Cygwin
@@ -59,1 +61,1 @@
-== Features in 3 minutes
+## Features in 3 minutes
@@ -61,1 +63,1 @@
-=== Cygwin
+### Cygwin
@@ -65,1 +67,1 @@
-=== Package manager
+### Package manager
@@ -69,1 +71,1 @@
-=== Shell
+### Shell
@@ -80,1 +82,1 @@
-=== Console
+### Console
@@ -84,1 +86,1 @@
-=== Proxying
+### Proxying
@@ -88,1 +90,1 @@
-=== Developer tools
+### Developer tools
@@ -99,1 +101,1 @@
-=== Plugin architecture
+### Plugin architecture
@@ -112,1 +114,1 @@
-=== Auto-update
+### Auto-update
@@ -118,1 +120,1 @@
-=== Installer
+### Installer
@@ -122,1 +124,1 @@
-== Using babun
+## Using babun
@@ -124,1 +126,1 @@
-=== Setting up proxy
+### Setting up proxy